templates/shared/forms/_recaptcha.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<script>
+  // move g-recaptcha-response to marketo field
+  var marketoForm = document.querySelector('form#mktoForm');
+  marketoForm.addEventListener(
+    'submit',
+    function() {
+      var input = document.createElement('input');
+      input.type = 'hidden';
+      input.name = "grecaptcharesponse";
```

Comment:
Do you think we should try something more like "lastReCAPTCHAUserResponse" just for consistency with the other solutions out there?

Reply:
Or would you rather not rock the boat, now that we've got it working?

Reply:
let's leave it for now.